examples/complete/atmos.yaml

```diff
@@ -128,3 +128,64 @@ commands:
         steps:
           - echo Playing ping-pong...
           - echo pong
+
+# Integrations
+integrations:
+
+  # Atlantis integration
+  # https://www.runatlantis.io/docs/repo-level-atlantis-yaml.html
+  atlantis:
+    # Path and name of the Atlantis config file `atlantis.yaml`
+    # Supports absolute and relative paths
+    # All the intermediate folders will be created automatically (e.g. `path: /config/atlantis/atlantis.yaml`)
+    # Can be overridden on the command line by using `--output-path` command-line argument in `atmos atlantis generate repo-config` command
+    # If not specified (set to an empty string/omitted here, and set to an empty string on the command line), the content of the file will be dumped to `stdout`
+    path: "atlantis.yaml"
+
+    # Config templates
+    # Select a template by using the `--config-template <config_template>` command-line argument in `atmos atlantis generate repo-config` command
+    config_templates:
+      config-template-1:
+        version: 3
+        automerge: true
+        delete_source_branch_on_merge: true
+        parallel_plan: true
+        parallel_apply: true
```

Comment:
If these keys are omitted, does `atmos` remove the keys from the outputted `atlantis.yaml` file?

Reply:
Ah this I don't like because that forces me to have `atmos` manage my `atlantis.yaml` file when I only want `atmos` to manage my `projects` and maybe my `workflows`.

The issue comes up if atlantis has an additional yaml key in a new release and I want to add it to my `atlantis.yaml` file, I have to put in a PR into atmos in order to add it.

It would be better to use a generic map that would allow any key to be added to the output mapping

Reply:
Ok, sounds good.

I created this in the backlog https://github.com/cloudposse/atmos/issues/190